wren-ai-service/docs/config_examples/config.open_router.yaml

```
@@ -0,0 +1,158 @@
+  enable_column_pruning: false
+  query_cache_maxsize: 1000
+  query_cache_ttl: 3600
+  langfuse_host: https://cloud.langfuse.com
+  langfuse_enable: true
+  logging_level: DEBUG
+  development: false
+  historical_question_retrieval_similarity_threshold: 0.9
+  sql_pairs_similarity_threshold: 0.7
+  sql_pairs_retrieval_max_size: 10
+  instructions_similarity_threshold: 0.7
+  instructions_top_k: 10
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)